spread-lxd.yaml

```diff
@@ -0,0 +1,45 @@
+# just a trivial grouping for resource definitions reused by all systems
+resoures:
+  common: &common-resources
+    mem: 4096MiB
+    cpu: 4
+    # root disk size
+    size: 15GiB
+
+# list of actual systems that are expected to match ones requested by spread
+system:
+  ubuntu-24.04-64:
+    image: ubuntu:24.04
+    # this is the default
+    vm: true
+    setup-steps: ubuntu
+    resources: *common-resources
+  ubuntu-22.04-64:
+    image: ubuntu:22.04
+    setup-steps: ubuntu
+    resources: *common-resources
+  ubuntu-20.04-64:
+    image: ubuntu:20.04
+    setup-steps: ubuntu
+    resources: *common-resources
+  ubuntu-core-24-64:
+    image: ubuntu:24.04
+    setup-steps: ubuntu
+    resources: *common-resources
+    # enable/disable secure boot
+    secure-boot: false
+
+# setup steps after a system has been allocated
+setup:
+  ubuntu:
+    # those are pulled out of spread
+    - cloud-init status --wait
+    - sed -i "s/^\s*#\?\s*\(PermitRootLogin\|PasswordAuthentication\)\>.*/\1 yes/" /etc/ssh/sshd_config
```

Comment:
This is a bit too magic for my eyes, a comment explaining what this does would help :)